helpers/blaze/range.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
**(non-blocking)** Is this rule, and the few others added in this PR doing anything? `eslint-disable-next-line` but nothing on the next line.

Reply:
Nice catch, will remove.